morphology/morph.py

```diff
@@ -0,0 +1,175 @@
+#! /usr/bin/python
+#
+# morph.py
+#
+# Usage:
+#     module for madlib.py
+#
+# Apply morphology to words
+#
+
+import sys
+import os
+import re
+
+# Irregular word .txt file dictionaries
+cwd = os.getcwd()
+nouns_filename = f"{cwd}/morphology/irreg_nouns.txt"
+verbs_filename = f"{cwd}/morphology/irreg_verbs.txt"
+artcls_filename = f"{cwd}/morphology/irreg_artcls.txt"
+
+# Open files and create python dictionaries
+irreg_nouns = {}
+try:
+    file_handle = open(nouns_filename, 'r')
+except:
+    print(f" ERROR: Failed to open words file '{nouns_filename}'")
+    print("  Exiting system from morphology/morph.py\n")
+    sys.exit()
+
+num_lines = 0
+while True:
+    num_lines += 1
+    line = file_handle.readline()
+    if not line:
+        # at end of file
+        break
+    # get rid of dashes
+    line = line.replace('–', '')
+    # split words in line
+    words = line.split()
+    irreg_nouns[words[0]] = words[1]
```

Comment:
You define a variable called `num_lines` and you update it but it is never actually used for anything.  Is it debug cruft?